src/strategy-management/Toolkit.ts

```diff
@@ -755,22 +754,12 @@ export class Toolkit {
       spreadPercentage
     );
 
-    const result = {
-      buyPriceLow: trimDecimal(buyPriceLow, quoteDecimals),
-      buyPriceHigh: trimDecimal(prices.buyPriceHigh, quoteDecimals),
-      buyPriceMarginal: trimDecimal(prices.buyPriceMarginal, quoteDecimals),
-      sellPriceLow: trimDecimal(prices.sellPriceLow, quoteDecimals),
-      sellPriceHigh: trimDecimal(sellPriceHigh, quoteDecimals),
-      sellPriceMarginal: trimDecimal(prices.sellPriceMarginal, quoteDecimals),
-      marketPrice: trimDecimal(marketPrice, quoteDecimals),
-    };
-
     logger.debug('calculateOverlappingStrategyPrices info:', {
       quoteDecimals,
```

Comment:
Looks like you don't need quoteDecimals anymore. You can remove it and make the function sync.

Reply:
good catch - but note that the method signature will change